### DISCUSSION TITLE
Improve error handling

### DISCUSSION THREAD

**Discussion by Hofer-Julian** [borgbase/vorta#840] [Ideas]
Currently Vorta displays output from borg only in small labels in the UI.
If borg outputs several lines at the same time it is impossible to read the whole message, without opening the log file.

Additionally, Vorta could give helpful suggestions with some common errors:
- [ ] locked repository (give the option to break-lock, included with a suitable warning https://github.com/borgbase/vorta/issues/374)
- [ ] archive is missing (e.g. pruned by another borg process)
- [ ] missing ssh key
- [ ] no network connection
- [ ] no (unix) permission

Both UI change and improved error handling should probably implemented at the same time as these issues are interconnected.

**Comment by ThomasWaldmann** [comment]
For borgweb I implemented coloured log output (it checks the log level INFO, WARNING, ERROR and chooses respective colours). Maybe a nice thing for vorta, too?

Besides "marking" borg's log output somehow, I would appreciate if users can fully see it, so do not remove stuff, only add nice hints.

**Comment by Wikinaut** [comment]
I just wanted to report the same problem:

The indicated archive ``/mnt/nas10/borgbackup```was already deleted a long time ago. After clicking "Auffrischen" (refresh) I briefly see a message (that the archive is not present any more "ERROR: archive ... does not exist") but this message is really important and should be "sticky".

In other words: to show "Auffrischen der Archive erledigt" is really misleading, when the Refresh was done but with an Error result.

 
![grafik](https://user-images.githubusercontent.com/1151915/77862729-9523f080-721d-11ea-8fed-1ea56b472a7e.png)

* Vorta 0.6.24
* Borg 1.1.10
